.azure-pipelines/js-mgmt-ci.yml

```diff
@@ -0,0 +1,53 @@
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    exclude:
```

Comment:
Is your plan to try and exclude the data-plane projects? I'm not sure how scalable/maintainable this approach is going to be. @mitchdenny has been working on a way to tag projects and we should try using the same mechanism.

Reply:
I was not aware of the tags. I'll follow up with Mitch.